exporter/terra/terra_listener.py

```diff
@@ -88,10 +112,15 @@ def get_consumers(self, _consumer: Type[Consumer], channel) -> List[Consumer]:
                                         callbacks=[self.experiment_message_handler],
                                         prefetch_count=1)
 
+        #data_consumer = _consumer([_TerraListener.queue_from_config(self.data_queue_config)],
+        #                                callbacks=[self.data_message_handler],
+        #                                prefetch_count=1)
+
```

Comment:
I know we are about to add this soon but might be good to clean up comments for now